### ISSUE TITLE
Xpack plugin

### ISSUE THREAD

**Issue by irizzant** [bitnami/bitnami-docker-fluentd#9]
**Description**

-------------------------------
    BUG REPORT INFORMATION
-------------------------------
elasticsearch-xpack gem is missing in the image, even if elasticsearch plugin is installed by default.

**Steps to reproduce the issue:**
1. start the image
1. verify the plugin list does not include elasticsearch-xpack gem

**Describe the results you received:**

elasticsearch-xpack gem is missing

**Describe the results you expected:**

elasticsearch-xpack gem should be available

**Version**

- Output of `docker version`:

```
Client: Docker Engine - Community
 Version:           19.03.12
 API version:       1.40
 Go version:        go1.13.10
 Git commit:        48a66213fe
 Built:             Mon Jun 22 15:45:36 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.12
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.10
  Git commit:       48a66213fe
  Built:            Mon Jun 22 15:44:07 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683

```

- Output of `docker info`:

```
Client:
 Debug Mode: false

Server:
 Containers: 2
  Running: 2
  Paused: 0
  Stopped: 0
 Images: 2
 Server Version: 19.03.12
 Storage Driver: overlay2
  Backing Filesystem: xfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 4.15.0-109-generic
 Operating System: Ubuntu 18.04.4 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 8
 Total Memory: 31.21GiB
 Name: pclnxdev22
 ID: D372:OWDS:SZT3:OLYW:XB77:TONZ:ELT5:QG4C:DJ3D:KKY6:2UCK:MRQB
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Registry Mirrors:
  https://maven-repo.sdb.it:18080/
 Live Restore Enabled: false

WARNING: No swap limit support

```

- Output of `docker-compose version` (if applicable):

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

**Comment by juan131** [comment]
Hi @irizzant 

Thanks so much for your feedback!! We just released a new version (**1.11.1-debian-10-r18**) including the [elasticsearch-xpack gem](https://rubygems.org/gems/elasticsearch-xpack):

```console
$ docker run --rm -it bitnami/fluentd:1.11.1-debian-10-r18 -- ls /opt/bitnami/fluentd/gems | grep elasticsearch
elasticsearch-7.8.0				multipart-post-2.1.1
elasticsearch-api-7.8.0				netrc-0.11.0
elasticsearch-transport-7.8.0			oj-3.3.10
elasticsearch-xpack-7.8.0			prometheus-client-0.9.0
fluent-plugin-elasticsearch-4.0.11		serverengine-2.2.1
```

**Comment by irizzant** [comment]
Hi @juan131 thank you
